client.go

```diff
@@ -24,10 +24,10 @@ import (
 const (
 	apiVersion                            = "1.1"
 	paramIntegration    jsonrpc.ParamName = "integration"
+	paramQuery          jsonrpc.ParamName = "query"
 	paramData           jsonrpc.ParamName = "data"
 	paramSignalsByInput jsonrpc.ParamName = "signalsByInput"
 	paramItemsBySignal  jsonrpc.ParamName = "itemsBySignal"
-	paramQuery          jsonrpc.ParamName = "query"
 	paramItems          jsonrpc.ParamName = "items"
 	paramCalculations   jsonrpc.ParamName = "calculations"
 	paramFormat         jsonrpc.ParamName = "format"
```

Comment:
maybe just order alphanumerically instead of trying to impose any logic.

There are some helpers in VS Code to sort selected lines.
![image](https://github.com/clarify/clarify-go/assets/75205/4c55ff10-1d13-406d-8397-5fa866dca055)